README.md

```diff
@@ -1,27 +1,186 @@
+    num_examples_per_instance=25,
+    gap=1,
+    verbose=True,
+)
+labels.head()
+```
+```
+          customer_id         cutoff_time  total_spent
+label_id
+0                   1 2014-01-01 04:13:51        65.11
+1                   1 2014-01-03 15:41:34       101.08
+2                   1 2014-01-05 11:46:10        16.78
+3                   1 2014-01-06 09:54:58       108.16
+4                   1 2014-01-08 08:54:02        48.33
+```
+
+### Transform Labels
+With the generated [`LabelTimes`](https://docs.compose.ml/en/latest/generated/composeml.LabelTimes.html#composeml.LabelTimes), we will apply specific transforms for our prediction problem. To make the labels binary, a threshold is applied for amounts exceeding $100.
+
+```python
+labels = labels.threshold(100)
+labels.head()
+```
+```
+          customer_id         cutoff_time  total_spent
+label_id
+0                   1 2014-01-01 04:13:51        False
+1                   1 2014-01-03 15:41:34         True
+2                   1 2014-01-05 11:46:10        False
+3                   1 2014-01-06 09:54:58         True
+4                   1 2014-01-08 08:54:02        False
+```
+
+Additionally, the label times are shifted one hour earlier for predicting in advance.
+
+```python
+labels = labels.apply_lead('1h')
+labels.head()
+```
 ```
+          customer_id         cutoff_time  total_spent
+label_id
+0                   1 2014-01-01 03:13:51        False
+1                   1 2014-01-03 14:41:34         True
+2                   1 2014-01-05 10:46:10        False
+3                   1 2014-01-06 08:54:58         True
+4                   1 2014-01-08 07:54:02        False
+```
+
+### Describe Labels
+
+After transforming the labels, we can use [`LabelTimes.describe`](https://docs.compose.ml/en/latest/generated/methods/composeml.LabelTimes.describe.html#composeml.LabelTimes.describe) to print out the distribution with the settings and transforms that were used to make these labels. This is useful as a reference for understanding how the labels were generated from raw data. Also, the label distribution is helpful for determining if we have imbalanced labels.
+
+```python
+labels.describe()
+```
+```
+Label Distribution
+------------------
+False      75
+True       50
+Total:    125
+
+
+Settings
+--------
+num_examples_per_instance    25
+minimum_data                 1h
+window_size                  1h
+gap                           1
+
+
+Transforms
+----------
+1. threshold
+  - value:    100
+
+2. apply_lead
+  - value:    1h
+```
+
+## Testing & Development
+The Feature Labs community welcomes pull requests. Instructions for testing and development are available here.
+
+## Support
+The Feature Labs open source community is happy to provide support to users of Compose. Project support can be found in four places depending on the type of question:
+
+1. For usage questions, use [Stack Overflow](https://stackoverflow.com/questions/tagged/composeml) with the `composeml` tag.
```

Comment:
On Stack Overflow, the `compose` tag has been used in unrelated questions. Should the tag still be `compose` or `composeml`?